deploy/ods-pipeline/values.kind.yaml

```diff
@@ -0,0 +1,103 @@
+global:
+  imageTag: latest
+  taskSuffix: ''
+
+##
+# setup chart
+##
+setup:
+  enabled: true
+  # General
+  serviceAccountName: 'pipeline'
+
+  # Cluster
+  consoleUrl: 'http://example.com'
+
+  # Pipeline Manager
+  pipelineManager:
+    storageProvisioner: ''
+    storageClassName: 'standard'
+    storageSize: '2Gi'
+    replicaCount: 1
+    image:
+      registry: localhost:5000
+      namespace: ods
+      repository: ods-pipeline-manager
+      pullPolicy: Always
+      tag: "latest"
+    resources:
+      limits:
+        cpu: 100m
+        memory: 128Mi
+      requests:
+        cpu: 100m
+        memory: 128Mi
+
+  # Notification Webhook
+  notification:
+    # notifications are disabled by default, i.e. the ConfigMap won't be installed
+    enabled: false
+    # URL of the configured webhook
+    url: 'http://example.com'
+    # The HTTP method to be used
+    method: 'POST'
+    # The HTTP content type header
+    contentType: 'application/json'
+    # Specify the outcomes you want to be notified of (allowed values: c.f.
+    # https://tekton.dev/docs/pipelines/pipelines/#using-aggregate-execution-status-of-all-tasks)
+    notifyOnStatus:
+      - 'Failed'
+    # Template to be processed and accepted by the configured webhook in use
+    # Below example might work for Microsoft Teams
+    requestTemplate: |-
+      {
+        "@type": "MessageCard",
+        "@context": "http://schema.org/extensions",
+        "themeColor": {{if eq .OverallStatus "Succeeded"}}"237b4b"{{else}}"c4314b"{{ end }},
+        "summary": "{{.ODSContext.Project}} - ODS Pipeline Run {{.PipelineRunName}} finished with status {{.OverallStatus}}",
+        "sections": [
+          {
+            "activityTitle": "ODS Pipeline Run {{.PipelineRunName}} finished with status {{.OverallStatus}}",
+            "activitySubtitle": "On Project {{.ODSContext.Project}}",
+            "activityImage": "https://avatars.githubusercontent.com/u/38974438?s=200&v=4",
+            "facts": [
+              {
+                "name": "GitRef",
+                "value": "{{.ODSContext.GitRef}}"
+              },
+              {
+                "name": "Environment",
+                "value": "{{.ODSContext.Environment}}"
+              }
+            ],
+            "markdown": true
+          }
+        ],
+        "potentialAction": [
+          {
+            "@type": "OpenUri",
+            "name": "Go to PipelineRun",
+            "targets": [
+              {
+                "os": "default",
+                "uri": "{{.PipelineRunURL}}"
+              }
+            ]
+          }
+        ]
+      }
+
+##
+# tasks chart
+##
+tasks:
+  enabled: true
+  # Default values for chart.
+  # This is a YAML-formatted file.
+  # Declare variables to be passed into your templates.
+
+  registry: localhost:5000
```

Comment:
If I'm not mistaken, this could also be set to GitHub's container registry. It would be nice to explain this option in the comments, as it allows to use images without building them locally ...

Reply:
Maybe I'm getting this wrong but I think we still need this, at least for the Github actions workflow, since the images on ghcr.io are only updated after the tests were run successfully?

Reply:
Oh sorry I think my wording is quite unclear.

Yes, I agree we need `registry: localhost:5000`. I just wanted to highlight that a user could override this locally and pull images directly from ghcr.io.

That said, let's just leave it like it is and add a comment later on if someone thinks this is useful.

Reply:
I'll add a comment to point this out more clearly